SensusService/Probes/PollingProbe.cs

```diff
@@ -208,7 +237,32 @@ protected override void InternalStart()
 
                     }, GetType().FullName + " Poll", TimeSpan.FromMinutes(_pollingTimeoutMinutes), userNotificationMessage);
 
+#if __IOS__
+                if (_significantChangePoll)
+                {
+                    _locationManager.RequestAlwaysAuthorization();
+                    _locationManager.DistanceFilter = 5.0;
+                    _locationManager.PausesLocationUpdatesAutomatically = true;
```

Comment:
I don't think we want this to be `true`. If location updates get paused, they will not resume until the user activates the app.